<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="INDP277LT" failures="1" tests="2" name="Test02" time="0.03" errors="0" timestamp="2021-01-09T02:51:52 IST">
  <testcase classname="listener.Demo2" name="testMethod03" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [3] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [3] but found [2]
at listener.Demo2.testMethod03(Demo2.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testMethod03 -->
  <testcase classname="listener.Demo2" name="testMethod04" time="0.005"/>
</testsuite> <!-- Test02 -->
